build/README.md

```diff
@@ -425,7 +473,13 @@ Run SDK server (sidecar) in test mode (which would record all GRPC requests) ver
 Clean leftover binary and package files after running SDK conformance tests.
 
 #### `make test`
-Run the linter and tests
+Run the go tests, the sdk tests, the website tests and yaml tests.
```

Comment:
does this also run the linter?

Reply:
It doesn't - I feel like it used to, but it was removed to solve some issues with the CI systems so it could be run separately.